frontend/test/metabase/scenarios/question/native.cy.spec.js

```diff
@@ -568,4 +568,42 @@ describe("scenarios > question > native", () => {
       cy.findAllByText("Gizmo").should("not.exist");
     });
   });
+
+  it.skip("should run with the default field filter set (metabase#15444)", () => {
+    cy.intercept("POST", "/api/dataset").as("dataset");
+
+    cy.visit("/");
+    cy.icon("sql").click();
+    cy.get(".ace_content").type("select * from products where {{category}}", {
+      parseSpecialCharSequences: false,
+    });
+    // Change filter type from "Text" to Field Filter
+    cy.get(".AdminSelect")
+      .contains("Text")
+      .click();
+    popover()
+      .findByText("Field Filter")
+      .click();
+    popover()
+      .findByText("Products")
+      .click();
+    popover()
+      .findByText("Category")
+      .click();
+    cy.findByText("Required?").scrollIntoView();
```

Comment:
Doesn't this need a click to enable-toggle it?
```suggestion
    cy.findByText("Required?").scrollIntoView().click();
```

Reply:
No. The point wasn't to toggle it. Just to scroll it into view so that Cypress can see the default filter input field which was hidden at the bottom of the page.